### PR TITLE
NU-1106 fix style in comment input

### PR DESCRIPTION
**Pull request by bohdanprog**
## Describe your changes
before:
image
after:
image
## Checklist before merge
- [ ] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Code formatting checked
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

### CODE REVIEW

File: designer/client/src/components/withFocus.tsx [hunk 1]
```diff
@@ -33,6 +33,10 @@ export function TextAreaWithFocus({ className, ...props }: TextAreaWithFocusProp
     return <textarea {...props} className={cx(withFocus, className)} />;
 }
 
+export const TextAreaNodeWithFocus = styled(TextAreaWithFocus)`
+    ${NodeInputCss}
```
Comment (JulianWielga) [inline designer/client/src/components/withFocus.tsx:37]: why not ```styled(TextAreaWithFocus)(NodeInputCss)```?

Comment (bohdanprog) [reply designer/client/src/components/withFocus.tsx]: fix

**Review by Dzuming — APPROVED** [review]

File: designer/client/cypress/e2e/process.cy.ts [hunk 2]
```diff
@@ -153,6 +153,18 @@ describe("Process", () => {
 
         it("should have counts button and modal", () => {
             cy.viewport("macbook-15");
+            cy.contains(/^deploy$/i).click();
```
Comment (Dzuming) [inline designer/client/cypress/e2e/process.cy.ts:156]: Can you explain to me all of this code? Why can't we just place `cy.get("[data-testid=window]").matchImage();` right before `cy.deployScenario();`?

Comment (bohdanprog) [reply designer/client/cypress/e2e/process.cy.ts:156]: inside deployScenario(); we open and close that window

Comment (Dzuming) [reply designer/client/cypress/e2e/process.cy.ts:156]: OK, so maybe we can paste an argument like, `withScreenshot`, and move this `cy.get("[data-testid=window]").matchImage();` inside of the `cy.deployScenario();` ?

Comment (bohdanprog) [reply designer/client/cypress/e2e/process.cy.ts]: fix

**Pull request merged**